internal/servers/controller/handlers/authmethods/oidc.go

```diff
@@ -269,7 +270,16 @@ func (s Service) authenticateOidcToken(ctx context.Context, req *pbs.Authenticat
 		}
 	}
 	if token == nil {
-		return nil, nil
+		attrs, err := structpb.NewStruct(map[string]interface{}{
+			"status": "unknown",
```

Comment:
perhaps define const for the field name?

Reply:
Done.